### PR TITLE
Add input file options for toggling optimisation and monitoring

### PR DESCRIPTION
**Pull request by dmcdougall**

### CODE REVIEW

File: src/stats/src/StatisticalInverseProblem.C [hunk 1]
```diff
@@ -256,11 +257,25 @@ StatisticalInverseProblem<P_V,P_M>::solveWithBayesMetropolisHastings(
   // user-provided initial seed, or use the user-provided seed for a
   // deterministic optimisation instead and seed the chain with the result of
   // the optimisation
-  if (this->m_seedWithMAPEstimator) {
+  if (this->m_seedWithMAPEstimator ||
+      m_optionsObj->m_ov.m_seedWithMAPEstimator) {
+    // Unfortunately, I didn't have the foresight to make this an input file
+    // option from the beginning, hence needing to check two flags
```
Comment (pbauman) [inline src/stats/src/StatisticalInverseProblem.C]: Why do you still need to keep both around?

**Comment by pbauman** [comment]
Otherwise :+1: after rebase.

File: src/stats/src/StatisticalInverseProblem.C (hunk 1 above)
Comment (dmcdougall) [inline src/stats/src/StatisticalInverseProblem.C]: Backwards compatibility.

**Comment by dmcdougall** [comment]
I've updated this to use the new options parsing foo.

**Comment by pbauman** [comment]
Source changes look OK. Looks like 448d56d failed `make check` though...

**Comment by dmcdougall** [comment]
That should be better.

**Pull request merged**